lib/alpha_taxonomy/taxon_linker.rb

```diff
@@ -0,0 +1,23 @@
+require "csv"
+
+module AlphaTaxonomy
+  class TaxonLinker
+    def run!
+      ImportFile.new.grouped_mappings.each do |base_path, taxon_titles|
+        taxon_content_ids = taxon_titles.map do |taxon_title|
```

Comment:
This lookup is perhaps better done by fetching all taxons outside of this loop (`#get_content_items(..)`) and then a `find` by title. Could save a lot of requests.